api-put-object-copy.go

```diff
@@ -29,12 +29,12 @@ import (
 )
 
 // CopyObject - copy a source object into a new object
-func (c Client) CopyObject(ctx context.Context, dst DestinationInfo, src SourceInfo) error {
+func (c Client) CopyObject(ctx context.Context, dst DestinationInfo, src SourceInfo) (UploadInfo, error) {
```

Comment:
This should be just ObjectInfo not UploadInfo

Reply:
> This should be just ObjectInfo not UploadInfo

Plan changed after discussion, it is UploadInfo now